### ISSUE TITLE
Allow specifying a custom inventory file

### ISSUE THREAD

**Issue by bonzini** [patchew-project/patchew#69]
Maybe this does not make much sense, but I am trying to learn ansible... Variable values can be placed in the inventory directory or in a separate extra_vars file. Maybe instead of -i/-s/-t (or in addition to it) it would make sense to pass an inventory file and scripts/deploy would automatically figure out the playbooks to run. The advantage would be that the inventory and the variables (e.g. IMAP user/password, etc.) could be kept together in version control...

**Comment by famz** [comment]
Yes this makes sense. `scripts/deploy` already generates a temporary inventory file. Providing a custom one should work just well. We'll then have to peek into inventory file for the host list, it's a bit ugly but simple enough. (It would be nice if ansible could figure out not to prompt for useless parameters in the first place, that way we can just do `playbooks = ["deploy.yml"]`.)

On the other hand, it is as easy to maintain a long `scripts/deploy` command in a version controlled script. I do this myself and I don't think it's any worse in comparison:

(password masked)

```
$ cat ~/bin/deploy-patchew-next-importer
./scripts/deploy -i root@qemu --become-user=root -e "
    instance_name=patchew-importer-next
    patchew_server=http://next.patchew.org
    importer_user=importer
    importer_pass=ihavepatches
    imap_server=imap.zoho.com
    imap_user=importer2@patchew.org
    imap_pass=xxxxxx
    imap_folders=INBOX
    imap_cert_fingerprint=ed66ca560a21501f43b24e54a2fd033c91c8b27e
    imap_delete_after_import=y"
```
